index.php

```diff
@@ -159,8 +159,30 @@
       </div>
     </div>
 </div>
+<?php
+  // Determine if "Query Types (over time)" should be shown
+  $queryTypesOverTime = false;
+  if(isset($setupVars['DASHBOARD_SHOW_QUERY_TYPES_OVER_TIME']))
```

Comment:
We test for the existence of the variable before trying to access it in order to silence warnings in `lighttpd`'s `error.log`